client/layout/transient-notices/index.js

```diff
@@ -19,7 +19,7 @@ const QUEUED_NOTICE_FILTER = 'woocommerce_admin_queued_notice_filter';
 
 function TransientNotices( props ) {
 	const { createNotice, removeNotice: onRemove } = useDispatch(
```

Comment:
I think we'll need to move the `createNotice` to this below `useDispatch` so it's using `core/notices2`.  Discussion in https://github.com/woocommerce/woocommerce-admin/pull/6809

Reply:
@joshuatf  That's a rather long discussion in #6809, from skimming it, I take it that CES uses notices2? I always thought notices2 was temporary. Is there a particular reason for using `notices2` though, we use `core/notices` everywhere else? And it appears we can [remove core/notices2 when WP version 5.9 is out](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/notices/README.md).

If you wanted to stick with `core/notices2` we could just do this:
`const { createNotice: createNotice2 } = useDispatch('core/notices2')` and make use of that within the `useEffect` for the queued notices.

I only noticed [this comment](https://github.com/woocommerce/woocommerce-admin/pull/6809#issuecomment-822921826) by @moon0326 for requiring notices2, but didn't test it myself as I am not too familiar with CES.

Reply:
> Is there a particular reason for using `notices2` though, we use `core/notices` everywhere else?

`notices2` was originally created to allow for the use of a forked version of the snackbar component with the following changes:

- Adding an icon
- Adding an ‘X’ to close the snackbar
- Making the snackbar not disappear automatically

I believe @becdetat ended up pushing all those changes upstream to Gutenberg, but I am not positive. Unfortunately, there are no followup comments on the P2 post (pbIJXs-r6-p2) about this to help easily figure out exact what ended up happening!

Reply:
Thanks for the insight @mattsherman, after you mentioning this it does look like they got merged: https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+author%3Abecdetat+is%3Aclosed it appears those changes are in versions 9.4 & 9.6, which [according to this ](https://developer.wordpress.org/block-editor/contributors/versions-in-wordpress/) would be in the Wordpress 5.7.1, which is mentioned in the link I mentioned above - https://github.com/woocommerce/woocommerce-admin/blob/main/packages/notices/README.md

Reply:
I could go with this route:

>  const { createNotice: createNotice2 } = useDispatch('core/notices2') and make use of that within the useEffect for the queued notices.

This way the enqueued notices could still use the above 3 features that Matt mentioned.

Reply:
@joshuatf I updated it to use `core/notices2` for the `createNotice` if you are able to test it again, and maybe also the functionality you added in #6809